plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -63,6 +63,16 @@
   bootdisk:
     description:
       - Enable booting from specified disk. C((ide|sata|scsi|virtio)\d+)
+  cipassword:
+    description:
+      - Cloud Init password of default user to create.
+    type: str
+    version_added: 1.2.0
```

Comment:
Why is this added in version 1.2.0 and not in 1.1.0?

Reply:
Because the chance that this will get merged in the next couple of hours is nil.

Reply:
needs to be changed to `1.3.0`